src/audio/base_fw.c

```diff
@@ -252,6 +263,15 @@ static uint32_t basefw_set_system_time(uint32_t param_id,
 	global_system_time_info.dsp_time.val_l = (uint32_t)(current_dsp_time);
 	global_system_time_info.dsp_time.val_u = (uint32_t)(current_dsp_time >> 32);
 
+	/* use default timestamp if 64bit is not enabled since 64bit is necessary for host time */
+	if (!IS_ENABLED(CONFIG_LOG_TIMESTAMP_64BIT))
+		return IPC4_SUCCESS;
```

Comment:
Never silently drop things on the floor. Log something.

Reply:
sure, good idea.